extensions/pkg/controller/worker/genericactuator/actuator.go

```diff
@@ -68,6 +69,7 @@ type genericActuator struct {
 // NewActuator creates a new Actuator that reconciles
 // Worker resources of Gardener's `extensions.gardener.cloud` API group.
 // It provides a default implementation that allows easier integration of providers.
+// If machine-controller-manager should not be managed then only the delegateFactory must be provided.
```

Comment:
```suggestion
// If machine-controller-manager should be managed then only the delegateFactory must be provided.
```

Reply:
Oops, I read it as 
// If machine-controller-manager should not be managed, then only, the delegateFactory must be provided.
😄